balancer/pickfirst/pickfirstleaf/pickfirstleaf.go

```diff
@@ -179,14 +198,15 @@ func (b *pickfirstBalancer) resolverErrorLocked(err error) {
 	// The picker will not change since the balancer does not currently
 	// report an error. If the balancer hasn't received a single good resolver
 	// update yet, transition to TRANSIENT_FAILURE.
-	if b.state != connectivity.TransientFailure && b.addressList.size() > 0 {
+	if b.connectivityState != connectivity.TransientFailure && b.addressList.size() > 0 {
```

Comment:
E.g. of above: I find the usages confusing here -- how do we know whether it's correct for this to be checking `connectivityState` or `concludedState`?